recipe/meta.yaml

```diff
@@ -52,12 +52,6 @@ requirements:
   run:
     - curl
```

Comment:
@czender, does NCO use `curl` the executable at runtime or just `libcurl`?

Reply:
Good question. NCO does not use the curl executable.
I don't think there would be a problem eliminating
the curl executable so long as the curl library was
available. TBH, I'm not sure whether NCO even requires
libcurl. It may be historical (i.e., it used to) or
it may be that wget or DAP (which NCO needs) depend
on libcurl. Shall I test whether NCO really requires
libcurl?

Reply:
I can take out all curl and we can see what happens.